### PR TITLE
Removes Height, Light Width and Line Color from Sensor

### PR DESCRIPTION
**Pull request by zsarnett**

### CODE REVIEW

File: src/panels/lovelace/cards/hui-sensor-card.js [hunk 1]
```diff
@@ -144,8 +140,8 @@ class HuiSensorCard extends EventsMixin(LitElement) {
 
   _calcPoints(history, hours, width, detail = 1) {
     const coords = [];
-    const margin = this._config.line_width;
-    const height = this._config.height - margin * 4;
+    const margin = 5;
+    const height = 100 - margin * 4;
```
Comment (bramkragten) [inline src/panels/lovelace/cards/hui-sensor-card.js]: If it is no longer a variable we don't have to calculate it?
```suggestion
    const height = 380;
```

File: src/panels/lovelace/cards/hui-sensor-card.js [hunk 2]
```diff
@@ -246,6 +242,7 @@ class HuiSensorCard extends EventsMixin(LitElement) {
           padding: 16px;
           position: relative;
           cursor: pointer;
+          --line-color: var(--accent-color, var(--primary-color));
```
Comment (balloob) [inline src/panels/lovelace/cards/hui-sensor-card.js]: This is not used? I also don't think that we should fallback.

File: src/panels/lovelace/cards/hui-sensor-card.js [hunk 3]
```diff
@@ -85,9 +81,9 @@ class HuiSensorCard extends EventsMixin(LitElement) {
             ${
               _line
                 ? svg`
```
Comment (balloob) [inline src/panels/lovelace/cards/hui-sensor-card.js]: Funny, prettier doesn't format HTML inside things tagged with `svg`

File: src/panels/lovelace/cards/hui-sensor-card.js (hunk 2 above)
Comment (zsarnett) [reply src/panels/lovelace/cards/hui-sensor-card.js]: You mean adding this here doesnt magically make it used... 🤦‍♂️

File: src/panels/lovelace/cards/hui-sensor-card.js (hunk 1 above)
Comment (zsarnett) [reply src/panels/lovelace/cards/hui-sensor-card.js]: Good Call. I didnt even see that. Thank you

File: src/panels/lovelace/cards/hui-sensor-card.js (hunk 3 above)
Comment (zsarnett) [reply src/panels/lovelace/cards/hui-sensor-card.js]: I updated this so everything looks like prettier handles html tag

Comment (zsarnett) [reply src/panels/lovelace/cards/hui-sensor-card.js]: Actually I just had the Idea to change SVG to HTML format that and change it back.

**Review by balloob — APPROVED** [review]

**Pull request merged**